src/tokens/colors/extended.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
it might be good to make this more obvious as `extended-wipro` or `wipro-extended`

Reply:
Good point. I'll change it to `wipro-extended`.